src/analyses/threadEscape.ml

```diff
@@ -75,14 +75,16 @@ struct
     else
       ctx.local
 
+  (* TODO: why is there no combine_assign? *)
+
```

Comment:
You are right that it probably would to have handling similar to `assign`, rather than identity from `IdentitySpec`.